erpnext/docs/user/manual/en/schools/setup/room.md

```diff
@@ -1,6 +1,17 @@
 # Room
 
+A classroom is a learning space, a room in which both children and adults learn about things. A room in an educational institute can be a Class room, a laboratory or a Examination hall.
```

Comment:
un-necessary! A person using ERPNext for schools will know what a classroom is!

You need to say:

A classroom is a space (room or lab) where you want to schedule courses or examinations.